catalog/samples/helloSwift3.swift

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
Apropos the discussion for #188 and #302 I would opt for a more descriptive name. There are two parts though: what the file is called and what the action is called (they need not be the same). But ideally both are descriptive. This is a minor nit for now but will come to the fore eventually.